airflow/providers/sqlite/hooks/sqlite.py

```diff
@@ -28,12 +28,12 @@ class SqliteHook(DbApiHook):
 
     conn_name_attr = 'sqlite_conn_id'
     default_conn_name = 'sqlite_default'
-    supports_autocommit = False
```

Comment:
It seems this value was hardcoded for `SqliteHook` so I'd say this may be a breaking change

Reply:
Happy to change it, but since the `DbApiHook` has the same, is it needed? Maybe needed just to be explicit?

Reply:
Ok, I checked the `DbApiHook`, sounds reasonable.

Reply:
[Here's the code over in `DbApiHook`](https://github.com/apache/airflow/blob/a6db84afe1b2824149f735105665da39c2d7b6db/airflow/hooks/dbapi_hook.py#L54).